### ISSUE TITLE
Custom dataset ?

### ISSUE THREAD

**Issue by Michaelwhite34** [NVlabs/nvdiffrecmc#7]

**Comment by jmunkberg** [comment]
Yes, this is straightforward. You can adapt the included data readers to your datasets: https://github.com/NVlabs/nvdiffrecmc/tree/main/dataset

We require camera poses and foreground segmentation masks (stored in the alpha channel of the images), and I would recommend 50+ images with nicely distributed views covering the object.
